js/app/modules/contentGroup/contentGroup.js

```diff
@@ -41,6 +39,31 @@ const ContentGroup = new Module.Class({
         'has-more-content': GObject.ParamSpec.boolean('has-more-content',
             'Has more content', 'Has more content',
             GObject.ParamFlags.READABLE),
+        /**
+         * Property: paginate
+         * Whether this content group should paginate content
+         *
+         * Use this property to turn on pagination. If this property is set,
+         * a 'See more' button will appear at the bottom of the content group,
+         * which will allow user to page in more results manually. The number
+         * of results that get paged in on each click is determined by
+         * cards-per-page
+         */
+        'paginate': GObject.ParamSpec.boolean('paginate',
+            'Paginate', 'Whether to paginate content',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
+            false),
+        /**
+         * Property: cards-per-page
+         * Number of cards to be displayed per page
+         *
+         * Default:
+         *   10
```

Comment:
Yep, I wanted to be as explicit as possible in doc comments in the past, but we can probably drop this ... autogenerating is the first big task on my list after the SpiderMonkey upgrade